package-lock.json

```diff
@@ -1,9513 +1,8 @@
 {
   "name": "rally-study-template",
   "version": "0.3.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Mh, weird :O Sorry to nitpick on this, but looks like you have an older version of NPM that's rewriting the lockfile to an old  (but still supported) version. Would you kindly update NPM to version 7, discard this commit, and update again the lockfile?

Reply:
> Mh, weird :O Sorry to nitpick on this, but looks like you have an older version of NPM that's rewriting the lockfile to an old (but still supported) version. Would you kindly update NPM to version 7, discard this commit, and update again the lockfile?

oh ffs, heh yeah I'll upgrade my npm. thanks :)